application/F3DStarter.cxx

```diff
@@ -539,7 +557,7 @@ class F3DStarter::F3DInternals
     this->UpdateInterdependentOptions();
   }
 
-  void UpdateTypedAppOptions(const std::map<std::string, std::string>& appOptions)
+  void UpdateTypedAppOptions(const F3DOptionsTools::OptionsDict& appOptions)
```

Comment:
changes in this method are purely performance/style changes, no functionnal changes